index.js

```diff
@@ -0,0 +1,69 @@
+//Include the express library
+const express = require("express");
+//Include the morgan middleware
+const morgan = require("morgan");
+//Include the cors middleware
+const cors = require("cors");
+//Include the database
+const contacts = require("./data/contacts");
+
+//Create a new express application
+const app = express();
+
+//Tell express we want to use the morgan library
+app.use(morgan("dev"));
+//Tell express we want to use the cors library
+app.use(cors());
+//Tell express to parse JSON in the request body
+app.use(express.json());
+
+//Start up our server
+const port = 3030;
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}/`);
+});
+
+// GET
+
+app.get("/contacts", (req, res) => {
+  // Gets the list of contacts
+  res.json(contacts);
+});
+
+app.get("/contacts/:id", (req, res) => {
+  // Gets the contact by id
+  const id = req.params.id;
+  const foundContact = contacts.contacts.find((c) => c.id === Number(id));
+  res.json(foundContact);
```

Comment:
`{ contact: foundContact }`
But also add the contacts meetings to the contact object